scripts/kurtosis/network_params.json

```diff
@@ -36,7 +37,8 @@
                 "--miner.extradata=The_Dev_version_of_builder",
                 "--builder.algotype=greedy",
                 "--metrics.builder",
-                "--metrics.expensive"
+                "--metrics.expensive",
+                "--maxpeers 0"
```

Comment:
I think maxpeers=0 may be dropped to have better overview for buuilders on which txs were used.
But Builder shall recognize it based on block processed

Reply:
Let's analyze the details later, we got graphs now so it'll be easier to compare different setups